ui/packages/catalog-ui-search/src/main/webapp/react-component/menu/menu.tsx

```diff
@@ -1,10 +1,53 @@
 import * as React from 'react'
+import styled from '../styles/styled-components'
 
 const CustomElements = require('../../js/CustomElements.js')
 const Component = CustomElements.registerReact('menu')
 
 const mod = (n: any, m: any) => ((n % m) + m) % m
 
+const MenuRoot = styled(Component)`
+  max-height: 25vh;
+  position: relative;
+`
+
+const after = `
+  ::after {
+    display: inline-block;
+    content: '\f00c';
+    font-family: FontAwesome;
+    font-style: normal;
+    position: absolute;
+    top: 50%;
+    right: 0px;
+    width: 2.275rem;
+    text-align: center;
+    transform: translateY(-50%);
+  }
+`
+
+const ItemRoot = styled.div<any>`
```

Comment:
what does adding `<any>` do?

Reply:
Gives access to non-theme props. I can add a type here.